tests/dashboards/verifyGCRemoteInstance_test.js

```diff
@@ -0,0 +1,170 @@
+const assert = require('assert');
+
+const { remoteInstancesHelper } = inject();
+
+Feature('Monitoring Mysql and Postgresql DB running on Google Cloud');
+
+const instances = new DataTable(['instance', 'instanceType', 'metric']);
+
+instances.add(['pgsql13', 'postgresql', 'pg_stat_database_xact_rollback']);
+instances.add(['pgsql12', 'postgresql', 'pg_stat_database_xact_rollback']);
+instances.add(['pgsql14', 'postgresql', 'pg_stat_database_xact_rollback']);
+instances.add(['pgsql11', 'postgresql', 'pg_stat_database_xact_rollback']);
+instances.add(['pgsql10', 'postgresql', 'pg_stat_database_xact_rollback']);
+instances.add(['pgsql96', 'postgresql', 'pg_stat_database_xact_rollback']);
+instances.add(['mysql57', 'mysql', 'mysql_global_status_max_used_connections']);
+instances.add(['mysql56', 'mysql', 'mysql_global_status_max_used_connections']);
+instances.add(['mysql80', 'mysql', 'mysql_global_status_max_used_connections']);
+
+// Mapping here to avoid datatables to add those details to test names in allure report
+const remoteInstance = {
+  pgsql13: remoteInstancesHelper.remote_instance.gc.gc_pgsql_13,
+  pgsql14: remoteInstancesHelper.remote_instance.gc.gc_pgsql_14,
+  pgsql12: remoteInstancesHelper.remote_instance.gc.gc_pgsql_12,
+  pgsql11: remoteInstancesHelper.remote_instance.gc.gc_pgsql_11,
+  pgsql10: remoteInstancesHelper.remote_instance.gc.gc_pgsql_10,
+  pgsql96: remoteInstancesHelper.remote_instance.gc.gc_pgsql_96,
+  mysql57: remoteInstancesHelper.remote_instance.gc.gc_mysql57,
+  mysql80: remoteInstancesHelper.remote_instance.gc.gc_mysql80,
+  mysql56: remoteInstancesHelper.remote_instance.gc.gc_mysql56,
+};
+
+function getInstance(key) {
+  return remoteInstance[key];
+}
+
+Before(async ({ I, settingsAPI }) => {
+  await I.Authorize();
+});
+
+Scenario(
+  'Increasing Scrape Interval to Rare for remote pgsql instances bug @not-ui-pipeline @gcp',
```

Comment:
Can it be moved to beforeEach block?

Reply:
I did this intentionally for visibility purposes as I recall the allure report we generate completely ignored the actions taken in beforeEach, which created confusion for me.